docs/content/dagster-cloud/developing-testing/branch-deployments/using-branch-deployments.mdx

```diff
@@ -0,0 +1,135 @@
+---
+title: Using Branch Deployments | Dagster Cloud
+description: Develop and test in the cloud.
+---
+
+# Using Branch Deployments with TODO
+
+<Note>This guide is applicable to Dagster Cloud.</Note>
+
+In this guide, we'll walk you through setting up [Branch Deployments](/dagster-cloud/developing-testing/branch-deployments) with a general continuous integration (CI) or git platform, using the [`dagster-cloud` CLI](/guides/dagster-cloud-cli).
+
+**If you use GitHub for version control**, check out the dedicated [Branch deployments with GitHub guide](/dagster-cloud/developing-testing/branch-deployments/using-branch-deployments-with-github).
+
+---
+
+## Prerequisites
+
+Utilizing Branch Deployments requires setting up two components: the Branch Deployment agent and CI platform. You'll need:
+
+- **Editor** or **Admin** permissions in Dagster Cloud
+- To install the [`dagster-cloud` CLI](/guides/dagster-cloud-cli)
+- The ability to run a new agent in your infrastructure
+- The ability to configure your CI platform
+
+---
+
+## Step 1: Generate a Dagster Cloud agent token
+
+In this step, you'll generate a token for the agent. The Dagster Cloud agent will use this to authenticate to the agent API.
+
+1. Sign in to your Dagster Cloud instance.
+2. Click the **user menu (your icon) > Cloud settings**.
+3. In the **Cloud settings** page, click the **Configure tokens** tab.
+4. Click the **+ Create agent token** button.
+5. After the token has been created, click **Reveal token**.
+
+Keep the token somewhere handy - you'll need it to complete the setup.
+
+---
+
+## Step 2: Create and configure an agent
+
+[SAME CONTENT FROM THE GITHUB ACTIONS GUIDE](/dagster-cloud/developing-testing/branch-deployments/using-branch-deployments-with-github#step-2-create-and-configure-an-agent)
```

Comment:
This section will be the same as the corresponding section in the GitHub guide. I'll add it here once that content is reviewed and given a stamp, so I don't need to update it twice.